Despite listing pyinotify as a dependency, it isn't actually referenced anywhere.

--- ./setup.cfg.orig	2023-10-10 22:07:11.502935586 -0400
+++ ./setup.cfg	2023-10-10 22:08:15.522236184 -0400
@@ -23,7 +23,6 @@
 	aionotify
 	icalendar
 	python-dateutil
-	pyinotify
 	PyGObject
 	toml
 	tzlocal
